_static/js/custom.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+    window.onload = function () {
+        var isApiPresent = document.getElementsByTagName('redoc').length;
+        if(isApiPresent) {
+            window.addEventListener("hashchange", function(){
+                var locationHash = location.hash.split('#')[1],
+                    elementToScroll = null;
+                if(locationHash.split('/')[0] !== 'section'&& locationHash.split('/')[0] !== 'tag' ) {
```

Comment:
Assign this value to a variable so you don't have to generate it twice.

Reply:
This is more like POC, it will be fixed in another ticket